source/platform/platforms/Allegro5Platform.cpp

```diff
@@ -21,7 +21,7 @@
 static Gwk::Input::Allegro      g_gworkInput;
 static ALLEGRO_EVENT_QUEUE*     g_event_queue = nullptr;
 static ALLEGRO_DISPLAY*         g_display = nullptr;
-static Gwk::String              gs_ClipboardEmulator;
+static Gwk::String              gs_clipboard;
```

Comment:
If Allegro is managing the clipboard state I don't think we need to keep this here anymore.